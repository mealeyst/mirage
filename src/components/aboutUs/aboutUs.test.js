import React from 'react'
import 'jest-styled-components'

import { AboutUs } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) AboutUs', () => {
  const createAboutUs = (props) => {
    return shallowWithTheme(<AboutUs {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createAboutUs())
    .toMatchSnapshot()
  })
})
