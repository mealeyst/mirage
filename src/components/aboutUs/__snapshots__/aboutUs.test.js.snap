// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) AboutUs matching the snapshot 1`] = `
.c0 .c1,
.c0 .c2 {
  text-align: center;
  text-transform: uppercase;
}

.c0 .c1 {
  margin: 0 auto;
  margin-bottom: 20px;
}

.c0 .c2 {
  color: #0073D1;
  margin-top: 40px;
  margin-bottom: 10px;
}

.c0 .c3 {
  text-align: center;
  line-height: 1.5;
  max-width: 450px;
  margin: 0 auto;
  font-size: 18px;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

.c0 .flexDiv {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  margin-bottom: 60px;
}

@media (max-width:958px) {
  .c0 .c1,
  .c0 .c2 {
    font-size: 3.8rem;
  }
}

<BaseAboutUs
  className="c0"
  constrained={true}
  footer="BECAUSE IT’S MORE FUN WHEN EVERYONE WINS!"
  header="We believe in simplifying the lives of parents and celebrating real life with kids."
  padding={true}
  sections={
    Array [
      Object {
        "body": "And we want to celebrate them! Quirks, preferences and all—with clothes that express who they are as individuals.",
        "title": "KIDS ARE AWESOME",
      },
      Object {
        "body": "We’ll make your life easier, so you can get back to being the best snack-schlepper, soccer-practice-chauffeur...",
        "title": "PARENTS ARE SUPERHEROES",
      },
      Object {
        "body": "You shouldn’t have to go to a million sites and stores to find awesome clothes you AND your kids will love.",
        "title": "SHOPPING SHOULD BE EASY",
      },
    ]
  }
/>
`;
